pkg/auto/forward.port.go

```diff
@@ -0,0 +1,161 @@
+package auto
```

Comment:
This file isn't named according to Go standards. Replace the first `.` with an `_`. And do this for any other file names similarly.

Reply:
Fixing.